Add HTTP proxy to datadog output (#9297)
influxdata/telegraf: plugins/outputs/datadog/README.md

  ## Write URL override; useful for debugging.
  # url = "https://app.datadoghq.com/api/v1/series"

>   ## Set http_proxy (telegraf uses the system wide proxy settings if it's is not set)

Reviewer: Typo. Same in `sampleConfig`
```suggestion
  ## Set http_proxy (telegraf uses the system wide proxy settings if it isn't set)
```
